openhands/server/session/agent_session.py

```diff
@@ -71,7 +73,6 @@ async def start(
                 'Session already started. You need to close this session and start a new one.'
             )
 
-        self.loop = asyncio.new_event_loop()
```

Comment:
Moved this to the constructor

Reply:
This might solve the issue reported in [slack](https://openhands-ai.slack.com/files/U07PWSVNWEL/F07P87D2FA6/image.png) just now. 🤔

Reply:
Yes.